src/components/editProperty/EditProperty.jsx

```diff
@@ -0,0 +1,300 @@
+import React, { useState } from 'react'
+import DashboardNav from '../../pages/dashboard/dashboardnav/DashboardNav'
+import Sidebar from '../sidebar/Sidebar'
+import Footer from '../footer/Footer'
+import './style.css'
+import { useParams } from "react-router-dom";
+import { PropertyFeature } from '../propertiesInput/propertyFeatures'
+import axios from 'axios'
+import { useSelector, useDispatch } from 'react-redux'
+import { editStateProperty } from '../../features/properties/adminProperties'
+import { useNavigate } from 'react-router-dom'
+
+const EditProperty = () => {
+
+  const {id} = useParams();
+
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  
+  const token = localStorage.getItem('token')
+    let config = {
+      "headers": {
+       "Authorization": token,
+      }
+    }
+
+  const initialValues = {
+    image_url : '',
+    title : '',
+    address : '',
+    state : '',
+    land_area : '',
+    purpose : '',
+    description : '',
+    year_of_build : '',
+    price: '',
+    no_of_bathrooms : '',
+    no_of_rooms: '',
+    no_of_store : '',
+    no_of_garage : ''
+  }
+
+  const [editProperty, setEditProperty] = useState({...initialValues});
+
+  const editHandler = (e) => {
+    setEditProperty({
+      ...editProperty, [e.target.name] : e.target.value
+    })
+  }
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+  
+
+      const response = await axios.put('http://localhost:4000/v1/agent/properties/' + id ,{...editProperty}, config)
+      dispatch(editStateProperty({
+        id,
+        ...editProperty
+      }))
+      navigate('/my-properties')
+      console.log(response.data)
+    } catch (error) {
+      console.log(error)
```

Comment:
You are suppose to toast this error message to let the user know something Is wrong, your user won't be checking your console to figure out there is a problem.